### PR TITLE
Completion of EventRegistration(s)Resource and more unit tests.

### PR DESCRIPTION
**Pull request by jshort**
I commented out the get portion of the EventRegistrationsIntegrationTest because it just hangs.  However, if you deploy this built code and do a couple POSTs of registrations and then a get on the same URI, you do, in fact, get back the list of registrations so I'm not sure what the delta is between that and the integration test.

### CODE REVIEW

File: dropwizard-app/src/test/java/org/coner/it/CreateRegistrationIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+        Response addRegistrationResponse1 = client.target(eventRegistrationsUri)
+                .request(MediaType.APPLICATION_JSON_TYPE)
+                .accept(MediaType.APPLICATION_JSON_TYPE)
+                .post(Entity.json(addRegistrationRequest1));
+
+        assertThat(addRegistrationResponse1.getStatus()).isEqualTo(HttpStatus.CREATED_201);
+        final String registrationId1 = UnitTestUtils.getEntityIdFromResponse(addRegistrationResponse1);
+
+        // Get Registrations for Event
+        /*Response getRegistrationsResponseContainer = client.target(eventRegistrationsUri)
```
Comment (jshort) [inline dropwizard-app/src/test/java/org/coner/it/CreateRegistrationIntegrationTest.java]: This is the portion of the test that hangs.  The .get() is never executed.

Comment (jshort) [inline dropwizard-app/src/test/java/org/coner/it/CreateRegistrationIntegrationTest.java]: Darn, the hardcoded version of dropwizard-client that we need is no longer in the remote repo :(.

File: dropwizard-app/src/test/java/org/coner/it/CreateRegistrationIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+
+        Response addRegistrationResponse1 = client.target(eventRegistrationsUri)
+                .request(MediaType.APPLICATION_JSON_TYPE)
+                .accept(MediaType.APPLICATION_JSON_TYPE)
+                .post(Entity.json(addRegistrationRequest1));
+
+        assertThat(addRegistrationResponse1.getStatus()).isEqualTo(HttpStatus.CREATED_201);
+        final String registrationId1 = UnitTestUtils.getEntityIdFromResponse(addRegistrationResponse1);
+
+        Client client2 = IntegrationTestUtils.buildClient(RULE);
```
Comment (jshort) [inline dropwizard-app/src/test/java/org/coner/it/CreateRegistrationIntegrationTest.java]: I'm not happy about it, but creating a new client object to make the subsequent requests avoids the hang.

Comment (carltonwhitehead) [inline dropwizard-app/src/test/java/org/coner/it/CreateRegistrationIntegrationTest.java]: Definitely not ideal, but this workaround will do. I'll leave a comment for posterity.

**Pull request merged**